src/pages/Cart/components/CartProducts.jsx

```diff
@@ -0,0 +1,92 @@
+import { Link } from 'react-router-dom';
+import React, { useEffect } from 'react';
+
+import './CartProducts.scss';
+
+const CartProducts = ({ data, changeCount, deleteCartData }) => {
+  const { cartId, eachPrice, koreanName, englishName, quantity, thumbnail } =
+    data;
+
+  const handleCount = e => {
+    fetch(`http://10.58.52.69:3000/carts?cart_id=${cartId}`, {
+      method: 'PATCH',
+      headers: {
+        Authorization: localStorage.getItem('TOKEN'),
+      },
+      body: JSON.stringify({
+        cartId: cartId,
+        quantity: parseInt(e.target.value),
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.message === 'LACK_OF_QUANTITY') {
+          alert('재고가 부족합니다.');
+        }
+      });
+    changeCount(e, data);
+  };
+
+  const handleDelete = () => {
+    fetch(`http://10.58.52.69:3000/carts?cart_id=${cartId}`, {
+      method: 'DELETE',
+      headers: {
+        Authorization: localStorage.getItem('TOKEN'),
+      },
+      body: JSON.stringify({
+        cartId: cartId,
+      }),
+    }).then(response => {
+      if (response.status === 204) {
+        deleteCartData(data);
+      }
+    });
+    deleteCartData(data);
+  };
+
+  const Price = eachPrice => {
+    return parseInt(eachPrice).toLocaleString();
+  };
+
+  const productQuantity = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].map((v, i) => i);
```

Comment:
어떤 의미를 가진 코드일까요?
```suggestion
  const productQuantity = Array(10).fill().map((v, i) => i);
```
이런 식으로도 쓸 수는 있겠네요.

Reply:
select에 선택할 물품 숫자를 지정하려고 하는 코드입니다. 일단 Array(10).fill()으로 수정했습니다!